Packs/CortexAttackSurfaceManagement/Integrations/CortexAttackSurfaceManagement/CortexAttackSurfaceManagement.py

```diff
@@ -138,16 +263,44 @@ def format_asm_id(formatted_response: List[dict]) -> List[dict]:
 
     if formatted_response:
         for entry in formatted_response:
-            if entry.get('asm_ids'):
-                entry['asm_ids'] = entry['asm_ids'][0]
+            if entry.get("asm_ids"):
+                entry["asm_ids"] = entry["asm_ids"][0]
 
     return formatted_response
 
 
-''' COMMAND FUNCTIONS '''
+def get_api_error(response: Response):
+    """Raises a formatted error based on the response from the base_error file from the server.
+
+    Args:
+        response: Response object from an API endpoint.
+    Raises:
+        NotFoundError: Exception for when an API endpoint
+            returns an error that does not have a corresponding error message.
+        ProcessingError: Exception for when an API endpoint returns an error message.
+    """
+    error_code, error_message, extra_message, rcs_err_msg = "", "", "", ""
+    try:
+        json_response = response.json()
+        error_code = json_response.get("reply", {}).get("err_code", {})
+        error_message = json_response.get("reply", {}).get("err_msg", {})
+        extra_message = json_response.get("reply", {}).get("err_extra", {})
+        rcs_err_msg = f"{error_message}. {extra_message}"
+        response.raise_for_status()
+    except requests.exceptions.HTTPError as err:
+        if "Forbidden" not in str(err):
```

Comment:
What happens in "Forbidden" is in str (for this one or below).  Will it still get raised?

Reply:
This is to handle the existing error that is raised in the `test-module` function.

Reply:
makes sense, thank you